### ISSUE TITLE
Useful resource to understand question 12 of the midterm

### ISSUE THREAD

**Issue by noranta4** [erodola/DLAI-s2-2021#11]
Look at this [link](https://playground.tensorflow.org/#activation=relu&batchSize=10&dataset=circle&regDataset=reg-plane&learningRate=0.03&regularizationRate=0&noise=0&networkShape=2&seed=0.30181&showTestData=false&discretize=false&percTrainData=50&x=true&y=true&xTimesY=false&xSquared=false&ySquared=false&cosX=false&sinX=false&cosY=false&sinY=false&collectStats=false&problem=classification&initZero=false&hideText=false) for a nice playground to experimentally find the solutions to all the points of Question 12.
Change the number of neurons and activation functions and launch the training to see what happens in every configuration.

At this other [link](https://share.streamlit.io/lucmos/demo-nn-expressivity/main/ui.py) you can find a similar playground to look at the function learned by a MLP for a regression problem.
